cruise/CruiseInformation.csv

```diff
@@ -167,3 +167,6 @@ EK20190624,R/V Elakha,2019-06-24T14:55:00,2019-06-24T19:00:00,ce02shsp_00013 dep
 EK20190703,R/V Elakha,2019-07-03T15:17:00,2019-07-03T21:00:00,ce02shsp_00014 deployment
 AT-42-12,R/V Atlantis,2019-06-02T00:00:00,2019-07-09T00:00:00,Cabled (rvdata) Visions 19
 WJ20190617,M/V Warren Jr.,2019-06-17T18:00:00,2019-06-20T00:04:00,Pioneer 12b glider and AUV deployment
+AR35-05,R/V Neil Armstrong,2019-08-02T00:00:00,2019-08-24T23:59:00,Irminger 6
+SKQ201920S,R/V Sikuliaq,2019-09-18T00:00:00,2019-10-05T23:59:00,Papa 7
+AR-38,R/V Neil Armstrong,2019-10-05T00:00:00,2019-10-15T23:59:00,Pioneer 13
```

Comment:
Irminger 6 is scheduled to return to port on 25 August.
Pioneer 13 is AR39 (I'm agnostic on if there should be a hyphen or not) and start date is 25 September (end date is correct).

Reply:
Whoops on start date for P13 - the 5th is start of leg 2.  Will update.  
As for cruise number for P13- the draft cruise plan has AR-38 (with the dash).

Reply:
I have implemented changes noted by Sheri